cishouseholds/pipeline/input_file_stages.py

```diff
@@ -132,6 +133,19 @@
     "source_file_column": "blood_test_source_file",
 }
 
+cis_digital_parameters = {
+    "stage_name": "survey_responses_version_digital_ETL",
+    "dataset_name": "survey_responses_digital",
+    "id_column": "visit_id",
+    "validation_schema": validation_schemas["cis_digital_validation_schema"],
+    "column_name_map": None,
+    "datetime_column_map": cis_digital_datetime_map,
+    "transformation_functions": [],
+    "sep": "|",
+    "cast_to_double_list": survey_response_cast_to_double,
+    "source_file_column": "survey_response_source_file",  # Double check
+}
+
```

Comment:
Will need to add a reference to the new dictionary in the list below, so that the stage gets generated